pom.xml

```diff
@@ -0,0 +1,180 @@
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>uk.gov.companieshouse</groupId>
+    <artifactId>company-accounts.api.ch.gov.uk</artifactId>
+    <version>unversioned</version>
+    <packaging>jar</packaging>
+    <name>company-accounts-api</name>
+    <description>Company accounts API to handle company accounts</description>
+
+
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.0.3.RELEASE</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+
+    <properties>
+        <!-- Encoding -->
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+        <!-- JDK Version -->
+        <jdk.version>1.8</jdk.version>
+
+        <!-- Dependency Versions -->
+        <spring-web.version>5.0.7.RELEASE</spring-web.version>
+
+        <!-- Maven and Surefire plugins -->
+        <maven-compiler-plugin.version>3.1</maven-compiler-plugin.version>
+        <maven-surefire-plugin.version>2.21.0</maven-surefire-plugin.version>
+        <maven-jar-plugin.version>2.4</maven-jar-plugin.version>
+        <junit-platform-surefire-provider.version>1.2.0</junit-platform-surefire-provider.version>
+
+        <!-- JUnit testing -->
+        <junit-jupiter-engine.version>5.2.0</junit-jupiter-engine.version>
+        <mockito-junit-jupiter.version>2.18.0</mockito-junit-jupiter.version>
+
+        <!-- Sonar -->
+        <sonar-maven-plugin.version>3.4.0.905</sonar-maven-plugin.version>
+        <sonar.host.url>${CODE_ANALYSIS_HOST_URL}</sonar.host.url>
+        <sonar.login>${CODE_ANALYSIS_LOGIN}</sonar.login>
+        <sonar.password>${CODE_ANALYSIS_PASSWORD}</sonar.password>
+
+        <!-- Repositories -->
+        <artifactoryResolveSnapshotRepo>libs-snapshot-local</artifactoryResolveSnapshotRepo>
+        <artifactoryResolveReleaseRepo>libs-release-local</artifactoryResolveReleaseRepo>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-web</artifactId>
+            <version>${spring-web.version}</version>
+        </dependency>
+
+        <!--  Test  -->
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <version>${junit-jupiter-engine.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-junit-jupiter</artifactId>
+            <version>${mockito-junit-jupiter.version}</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+
+    <profiles>
+        <!-- release-publish is the profile that is activated when the build server builds and
+             publishes a release project to the cloud artifactory -->
```

Comment:
Do these comments need to be here (applies to the comments on the other profiles too)

Reply:
Yeh, I put these there as people where asking what the difference between the profiles were. This hopefully clarifies it a bit